src/ServiceControl.TransportAdapter/FailedMessageForwarder.cs

```diff
@@ -72,11 +72,11 @@ Task OnRetryMessage(MessageContext context)
 
         }
 
-        static Task Forward(Dictionary<string, string> newHeaders, MessageContext context, IDispatchMessages forwarder, string destination)
+        static async Task Forward(Dictionary<string, string> newHeaders, MessageContext context, IDispatchMessages forwarder, string destination)
```

Comment:
@SzymonPobiega Why was this change made?

Reply:
Crap. Only for debugging  and forgot to revert it. Good catch! ❤️